src/main/cpp/native_glue.cc

```diff
@@ -157,16 +159,17 @@ namespace litecore {
                   _jbytes(jbytes),
                   _critical(critical) {
             if (jbytes == nullptr) {
-                _slice = nullslice;
+                _slice = kFLSliceNull;
                 return;
             }
 
             jboolean isCopy;
```

Comment:
what are we doing with this `jboolean isCopy;`? Seems like it is unused

Reply:
I don't think it's necessary, but it's out of scope of the purpose of this PR.

Reply:
I will remove it. Thanks.